config.go

```diff
@@ -553,3 +564,66 @@ func (b *Builder) Domainname() string {
 func (b *Builder) SetDomainname(name string) {
 	b.Docker.Config.Domainname = name
 }
+
+// Change Translates the Dockerfile syntax commands and then saves into the buildah struct
+func (b *Builder) Change(option string) error {
+	opt := strings.SplitN(option, " ", 2)
+	if len(opt) != 2 {
+		return fmt.Errorf("Change option %q must be separated by ' '", option)
+	}
+	switch strings.ToUpper(opt[0]) {
+	case "ANNOTATION":
+		annotation := strings.SplitN(opt[1], "=", 2)
+		if len(annotation) > 1 {
+			b.SetAnnotation(annotation[0], annotation[1])
+		} else {
+			b.UnsetAnnotation(annotation[0])
+		}
+	case "ARCH":
+		b.SetArchitecture(opt[1])
+	case "ENTRYPOINT":
+		entrypointSpec, err := shellwords.Parse(opt[1])
+		if err != nil {
+			return fmt.Errorf("error parsing --entrypoint %q: %v", opt[1], err)
+		}
+		b.SetEntrypoint(entrypointSpec)
+	case "ENV":
+		env := strings.SplitN(opt[1], "=", 2)
+		if len(env) == 1 {
+			env = strings.SplitN(opt[1], " ", 2)
+		}
+		if len(env) > 1 {
+			b.SetEnv(env[0], env[1])
+		} else {
+			b.UnsetEnv(env[0])
+		}
+	case "CMD":
+		cmdSpec, err := shellwords.Parse(opt[1])
+		if err != nil {
+			return fmt.Errorf("error parsing --cmd %q: %v", opt[1], err)
+		}
+		b.SetCmd(cmdSpec)
+	case "CREATED-BY":
+		b.SetCreatedBy(opt[1])
+	case "EXPOSE":
+		b.SetPort(opt[1])
+	case "LABEL":
+		label := strings.SplitN(opt[1], "=", 2)
+		if len(label) > 1 {
+			b.SetLabel(label[0], label[1])
+		} else {
+			b.UnsetLabel(label[0])
+		}
+	case "MAINTAINER":
+		b.SetMaintainer(opt[1])
+	case "OS":
+		b.SetOS(opt[1])
+	case "USER":
+		b.SetUser(opt[1])
+	case "VOLUME":
+		b.AddVolume(opt[1])
+	case "WORKINGDIR":
+		b.SetWorkDir(opt[1])
+	}
+	return nil
```

Comment:
Do you need a 1 to 1 correspondence with Dockerfile commands here?  For instance are Add, Healthcheck, Shell, Copy and others needed?